pkg/inventory/inventorycm.go

```diff
@@ -34,182 +27,76 @@ var ConfigMapGVK = schema.GroupVersionKind{
 // WrapInventoryObj takes a passed ConfigMap (as a resource.Info),
 // wraps it with the ConfigMap and upcasts the wrapper as
 // an the Inventory interface.
-func WrapInventoryObj(inv *unstructured.Unstructured) Storage {
-	return &ConfigMap{inv: inv}
+func WrapInventoryObj(inv *unstructured.Unstructured) Inventory {
+	internalInventory, err := configMapToInventory(inv)
+	if err != nil {
+		panic(err) //TODO: fix
```

Comment:
Seems weird that a function called `WrapInventoryObj` could error. What's the case for this?

Reply:
Sent https://github.com/kubernetes-sigs/cli-utils/pull/659